neighsyncd/restore_neighbors.py

```diff
@@ -33,8 +33,9 @@
 # timeout the restore process in 1 min if not finished
 # This is mostly to wait for interfaces to be created and up after system warm-reboot
 # and this process is started by supervisord in swss docker.
-# It would be good to keep that time below routing reconciliation time-out.
-TIME_OUT = 60
+# It would be good to keep that time consistent routing reconciliation time-out, here
+# we are upstream, we shouldn't give up before down stream.
+TIME_OUT = 110
```

Comment:
>110 [](start = 11, length = 3)

The same. Are they the same thing?

Reply:
The BGP timeout is 120 seconds, Zhenggen want this value to be slightly smaller than the BGP timeout. And he also requested to check if the bgp timeout is set in the state DB. I added the code below.

Reply:
Updated comments.